Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -147,11 +204,13 @@ static MTLStoreAction storeAction(WGPUStoreOp storeOp)
     if (!validateRenderPassDescriptor(descriptor))
         return RenderPassEncoder::createInvalid(m_device);
 
-    MTLRenderPassDescriptor* mtlDescriptor = [MTLRenderPassDescriptor renderPassDescriptor];
+    MTLRenderPassDescriptor* mtlDescriptor = [MTLRenderPassDescriptor new];
```

Comment:
this change is fine, the two methods are equivalent

Reply:
Right. I slightly prefer `new` rather than the other one, because the other one creates an autoreleased object, whereas ARC will release the `new`'ed object at the end of the lexical scope. So the lifetime is more tightly bounded.

But in reality there is probably no measurable difference